obal/data/playbooks/verify-tag/metadata.obal.yaml

```
@@ -0,0 +1,8 @@
+help: Verify packages against a tag
+variables:
+  tag:
+    help: Tag to verify against
```

with https://github.com/theforeman/obsah/pull/6 merged, you can now do:

```suggestion
  tag:
    parameter: tag
    help: Tag to verify against
```

and you'll get a positional argument

mind you, positional arguments are required, so you'll probably want to drop the `verify_all` arg and specialcase on an `all` tag or something

---

Can you release a new Obsah to make testing easier? Also, what does ` specialcase on an all tag or something` mean?

---

https://github.com/theforeman/obal/pull/243/files#r486301398 + https://github.com/theforeman/obal/pull/243/files#r486301876